src/components/button-minor/button-minor.test.js

```diff
@@ -0,0 +1,413 @@
+          "have.css",
+          "background",
+          `${destructive} none repeat scroll 0% 0% / auto padding-box border-box`
+        )
+        .and("have.css", "border-color", transparent)
+        .and("have.css", "color", "rgb(255, 255, 255)");
+      buttonDataComponent()
+        .eq(positionOfElement("second"))
+        .should(
+          "have.css",
+          "background",
+          `${transparent} none repeat scroll 0% 0% / auto padding-box border-box`
+        )
+        .and("have.css", "border-color", destructive)
+        .and("have.css", "color", destructive);
+      buttonDataComponent()
+        .eq(positionOfElement("third"))
+        .should(
+          "have.css",
+          "background",
+          "rgba(0, 0, 0, 0) none repeat scroll 0% 0% / auto padding-box border-box"
+        )
+        .and("have.css", "border-color", transparent)
+        .and("have.css", "color", destructive);
+    });
+
+    it.each(["noopener", "noreferrer", "opener"])(
+      "should render Button Minor with rel prop set to %s",
+      (rel) => {
+        CypressMountWithProviders(<ButtonMinor rel={rel}>Foo</ButtonMinor>);
+
+        buttonDataComponent().should("have.attr", "rel", rel);
+      }
+    );
+
+    it.each(["_blank", "_self", "_parent", "_top"])(
+      "should render Button Minor with target prop set to %s",
+      (target) => {
+        CypressMountWithProviders(<ButtonMinorCustom target={target} />);
+
+        buttonDataComponent().should("have.attr", "target", target);
+      }
+    );
+
+    it.each(["add", "share", "tick"])(
+      "should render Button Minor with type prop set to %s",
+      (type) => {
+        CypressMountWithProviders(<ButtonMinorCustom type={type} />);
+
+        buttonDataComponent().should("have.attr", "type", type);
+      }
+    );
+  });
+
+  describe("check events for Button Minor component", () => {
+    let callback;
+
+    beforeEach(() => {
+      callback = cy.stub();
+    });
+
+    it("should call onClick callback when a click event is triggered", () => {
+      CypressMountWithProviders(<ButtonMinorCustom onClick={callback} />);
+
+      buttonDataComponent()
+        .click({ force: true })
+        .then(() => {
+          // eslint-disable-next-line no-unused-expressions
+          expect(callback).to.have.been.calledOnce;
+        });
+    });
+
+    it("should call onBlur callback when a blur event is triggered", () => {
+      CypressMountWithProviders(<ButtonMinorCustom onBlur={callback} />);
+
+      buttonDataComponent()
+        .focus()
+        .blur({ force: true })
+        .then(() => {
+          // eslint-disable-next-line no-unused-expressions
+          expect(callback).to.have.been.calledOnce;
+        });
+    });
+
+    it("should call onKeyDown callback when a keydown event is triggered", () => {
+      CypressMountWithProviders(<ButtonMinorCustom onKeyDown={callback} />);
+
+      buttonDataComponent()
+        .focus()
+        .realPress("ArrowRight")
```

Comment:
can we pass few keys to check instead of 1?

Reply:
Added ArrowLeft, ArrowUp, ArrowDown and Enter to test.